dashboard/apps/prototype/static/js/project.js

```diff
@@ -1,43 +1,48 @@
-import Cookies from 'js-cookie';
 import 'whatwg-fetch';
 import URI from 'urijs';
 import moment from 'moment';
-import _ from 'lodash';
 import Plotly from './plotly-custom';
 
 /**
  * send a POST request to the backend to retrieve project profile
  */
-export function getProjectData(id) {
-  return fetch('/project.json', {
+export function getProjectData(id, csrftoken) {
+  const init = {
     credentials: 'same-origin',
     method: 'POST',
     headers: {
-      'X-CSRFToken': Cookies.get('csrftoken'),
+      'X-CSRFToken': csrftoken,
       'Accept': 'application/json',
       'Content-Type': 'application/json'
     },
     body: JSON.stringify({projectid: id})
-  }).then(response => response.json());
+  };
+  return fetch('/project.json', init)
+    .then(response => response.json());
 }
 
 /**
  * parse the financial infomation about the project
  */
 export function parseProjectFinancials(financial) {
-  let [months, costs] = _(financial).toPairs().sort().unzip().value();
-  months = months.map(m => moment(m, 'YYYY-MM').format('MMM YY'));
+  const _months = Object.keys(financial).sort();
```

Comment:
i decided to throw out `lodash` in the end. es6 has a lot of good native ways of doing things elegantly. this function for example, is not much longer using pure js.